scraper/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
```

Comment:
Cool stuff! 

Could you please add a readme for this scraper? Also about how to use it, if possible, thanks!